tests/problems/time/test_temporal_problem.py

```diff
@@ -135,6 +136,30 @@ def test_apoptosis_key_pipeline(self, adata_time: AnnData):
         problem.apoptosis_key = "new_apoptosis"
         assert problem.apoptosis_key == "new_apoptosis"
 
+    @pytest.mark.fast()
+    @pytest.mark.parametrize("c", [0.1, 1, 4])
+    def test_proliferation_key_c_pipeline(self, adata_time: AnnData, c: float):
+        keys = np.sort(np.unique(adata_time.obs["time"].values))
+        adata_time = adata_time[adata_time.obs["time"].isin([keys[0], keys[1]])]
+        delta = keys[1] - keys[0]
+        problem = TemporalProblem(adata_time)
+        assert problem.proliferation_key is None
+
+        problem.score_genes_for_marginals(gene_set_proliferation="human", gene_set_apoptosis="human")
+        assert problem.proliferation_key == "proliferation"
+
+        problem = problem.prepare(time_key="time", marginal_kwargs={"c": c})
+
+        expected_marginals = np.exp(
```

Comment:
Split this into multiple lines please (e.g., by assigning `subset = adata_time[adata_time.obs["time"] == keys[0]`), the automated formatting is bad.